### ISSUE TITLE
[BUG | Mod Compatibility] BBB causes biome-gradient-affected blocks to not render

### ISSUE THREAD

**Issue by Delfite** [FionaTheMortal/Better-Biome-Blend#62]
MC Version: 1.12.2
Forge Version: 14.23.5.2860
Mod Version: 1.1.7
Potential Cause(s): Unknown

Describe the bug: Blocks that are affected by a biome's color fail to render and cause a significant drop in FPS and constant lag spikes, this only occurs when you try to render the blocks. Otherwise, if there is a mod like Nothirium installed that allows for block-face culling then the lag isn't as significant so long as line-of-sight with the affected blocks remains broken. CCL errors show up in chat every few seconds no matter what I'm doing and a TON of errors will show up in the latest.log, resulting in my log file being 94MB in size. Disabling your mod fixed all of the issues listed. Disabling Optifine did not make the issue any better.

For convenience sake, the [Latest.log](https://gnomebot.dev/paste/1015701656354172958) has been formatted for easier readability. I'll also be uploading the modpack folder so you can download it through Curseforge with the exception of [PortalSuperCache version 0.2](https://github.com/LucunJi/PortalSuperCacheForge112/releases/tag/Alpha-0.2), which needs to be downloaded through its github repo.

The Modpack.zip contains all logs, crash reports, saves, mods (excluding portal super caching), in-game screenshots, as well as options.txt
[Modpack.zip](https://github.com/FionaTheMortal/better-biome-blend/files/9483260/Modpack.zip)

Note: Before I started adding more mods to my pack, BBB was working perfectly fine nor was it causing me any lag whatsoever as it's supposed to, so whatever I added later on is what's causing your mod to flip out. Xaero's Minimap was also throwing errors in the latest.log because of BBB.

**Comment by Delfite** [comment]
@FionaTheMortal

**Comment by FionaTheMortal** [comment]
Hopefully fixed. Difficult to validate with vague information. Only took 4 years ezpz.

**Comment by Delfite** [comment]
> Hopefully fixed. Difficult to validate with vague information. Only took 4 years ezpz.

Yeah, sorry about that. 18 year-old me wasn't very good (or thorough) with his bug reports back then. I would've updated this issue to be more specific about the problem, but I forgot I even made this report in the first place. Whadda world!
